src/constants.js

```diff
@@ -193,25 +184,14 @@ export const SAUCE_CONNECT_CLI_PARAMS = [{
     name: 'max-logsize',
     description: 'Rotate logfile after reaching <bytes> size. Disabled by default.',
     type: 'number'
-}, {
-    alias: 'M',
-    name: 'max-missed-acks',
-    description: 'The maximum amount of keepalive acks that can be missed before the client will trigger a reconnect. (default 30)',
-    type: 'number',
-    default: 30
 }, {
     name: 'metrics-address',
-    description: 'host:port for the internal web server used to expose client side metrics. (default "localhost:8888")'
+    description: 'host:port for the internal web server used to expose client side metrics. (default "localhost:8888")',
+    deprecated: true
```

Comment:
In this case, why is it deprecated?

Reply:
Updating SC docs as well - https://github.com/saucelabs/sauce-docs/pull/1222